README.md

```diff
@@ -4,6 +4,12 @@ MS-AMP is an automatic mixed precision package for deep learning developed by Mi
 
 Features:
 
+- Support O1 optimization: Apply FP8 to weights and weight gradients and support FP8 in communication.
+- Support O2 optimization: Support FP8 for two optimizizers(Adam and AdamW).
```

Comment:
two optimizizers(Adam and AdamW). -> two optimizers (Adam and AdamW).

Reply:
Fixed.